src/tfr_visualization/package.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>tfr_visualization</name>
+  <version>0.0.0</version>
+  <description>The tfr_visualization package</description>
```

Comment:
More descriptive description would be nice so we know exactly what goes in this package? Is it just rviz scripts? Any nodes that involve visualization even if they're custom written?

Reply:
I just forgot to add it, I'll elaborate.